report/6.Conclusion/Perspectivation.tex

```diff
@@ -1 +1,103 @@
+This limitation means that at a distance, hitting a small object is simply not viable due to the limitations of the motors.
+Likewise, the ability to predict the next location of an object was relinquished as it was not deemed necessary when not firing a projectile.
+Thus, 2 of the 4 \textit{should have} requirements have been fulfilled.
+
+\subsubsection{Could have and won't have}
+From the \textit{could have} section, the following requirements are formulated:
+\begin{itemize}
+	\item Ability to calculate the distance to the target object
+	\item Calculate the angle and power necessary to hit the target with a projectile
+\end{itemize}
+
+And in the \textit{won't have}:
+\begin{itemize}
+	\item Ability to actually take down targets
+	\item Ability to differentiate between the type of object that is being observed
+\end{itemize}
+
+None of these requirements have been fulfilled, as they are largely related to firing a projectile.
+
+\subsubsection{Conclusion}
+In conclusion, all of the must have requirements and half of the should have requirements have been fulfilled.
+This is enough to deem the project successful, as most of the unfulfilled requirements are related to firing a projectile, which was excluded from the scope of the project. 
+
+\subsection{Test cases}
+Along with the MoSCoW analysis, four test cases were presented: 
+\begin{itemize}
+    \item Dropping a balloon
+    \item Rolling a ball
+    \item Throwing a ball
+    \item Shooting a balloon
+\end{itemize}
+
+The first case, dropping a ball, was already presented in the MoSCoW conclusion and deemed successful.
+The test was conducted with a heart shaped, red balloon with a diameter of 30 centimeters, which was dropped at a distance of 4 meters and a height of 2 meters. 
+\texttt{F.L.A.T.} was able to keep it's laser on the target most of the time during the fall. 
+For the rolling a ball test case, \texttt{F.L.A.T.} was able to roughly keep it's laser on a ball with a diameter of 52 mm successfully. 
+However, the final two cases with a throwing a ball and shooting a balloon were not accomplished, as the device was not fast enough to process the speed of the object, and does not have the capabilities to shoot a balloon.
+
+\subsection{Machine intelligence}
+Several observations were made during the development of the MI aspect of the solution.
+Initially, the idea for the project was to recognize objects in real-time using machine learning, however as described in \autoref{des:sec:performance}, the object recognition algorithms available, were not performant enough given the hardware limitations.
+Instead, the variable motor power problem, described in \autoref{subsec:variable_motor_power}, turned out to be an excellent way to utilize MI for the solution.
+The calibration module was implemented using a neural network, which was executed on the NXT.
+This proved to be an excellent solution to the variable motor power problem, and a great way to use MI in the solution.
+
```

Comment:
```suggestion
The limitations of the solution was largely due to the limitations of the hardware.
A more powerful CPU would make the image-recognition feasible, and this would change the outcome of the system.
In a real world situation, object classification would be important, as actual targets are more difficult to track than red balloons, which have pretty simple attributes.
However, in a real world environment, the hardware would be be more powerful, and in turn making object classification possible.
```

Reply:
The solution was primarily limited by the hardware used. Linje 83